testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -557,6 +562,15 @@ def test_Cluster_add_metadata(self, cluster):
         cluster.add_metadata('test', metadata)
         assert np.all(cluster.metadata['test'] == metadata), \
                      "Cluster metadata isn't as expected"
+  
+    def test_empty_Cluster(self, empty):
+        assert empty.size == 0 and np.size(empty.elements) == 0 \
```

Comment:
Just for clarity would you be able to split this into 4 seperate assertions? I know it takes up more space, but it is much easier to see which one actually failed and easier to read with 4 seperate ones.